<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralarea">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="KMessageWidget" name="topMessageWidget"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="graphPage">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="welcomePage">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Codethink::lvtqtw::WelcomeScreen" name="welcomeWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="graphLoadProgress"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionGenerate_Database"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionParse_Codebase"/>
    <addaction name="actionCode_Generation"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionToggle_Split_View"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionClose_Current_Tab"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionSvg"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionProjectSettings"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDump_usage_log"/>
    <addaction name="actionReset_usage_log"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExport"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="objectHierarchyDoc">
   <property name="windowTitle">
    <string>Object Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Packages</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="packagesFilter">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Filter...</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Codethink::lvtqtw::TreeView" name="packagesTree">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Namespaces</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="namespaceFilter">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Filter...</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Codethink::lvtqtw::NamespaceTreeView" name="namespaceTree">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="usesInTheInterfaceDock">
   <property name="windowTitle">
    <string>Uses in the interface</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="usesInTheInterfaceTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="methodsDock">
   <property name="windowTitle">
    <string>Methods</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="methodsTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="usesInTheImplDock">
   <property name="windowTitle">
    <string>Uses in the implementation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="usesInTheImplTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="fieldsDock">
   <property name="windowTitle">
    <string>Fields</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="fieldsTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="providersDock">
   <property name="windowTitle">
    <string>Providers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="providersTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="clientsDock">
   <property name="windowTitle">
    <string>Clients</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="clientsTable">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="errorDock">
   <property name="windowTitle">
    <string>Error List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="Codethink::lvtqtw::ErrorView" name="errorView"/>
  </widget>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionToggle_Split_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Split View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCode_Generation">
   <property name="text">
    <string>Generate Code</string>
   </property>
  </action>
  <action name="actionParse_Codebase">
   <property name="text">
    <string>Parse Additional Source Code</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClose_Current_Tab">
   <property name="text">
    <string>Close Current Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGenerate_Database">
   <property name="text">
    <string>New Project From Source Code...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSvg">
   <property name="text">
    <string>Svg</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDump_usage_log">
   <property name="text">
    <string>Dump usage log</string>
   </property>
   <property name="toolTip">
    <string>Dump all the debug output of this current session to a file</string>
   </property>
  </action>
  <action name="actionReset_usage_log">
   <property name="text">
    <string>Reset usage log</string>
   </property>
   <property name="toolTip">
    <string>Free the in-memory log file, releasing memory.</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Codethink::lvtqtw::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ct_lvtqtw_treeview.h</header>
  </customwidget>
  <customwidget>
   <class>Codethink::lvtqtw::NamespaceTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ct_lvtqtw_namespacetreeview.h</header>
  </customwidget>
  <customwidget>
   <class>Codethink::lvtqtw::CycleTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ct_lvtqtw_cycletreeview.h</header>
  </customwidget>
  <customwidget>
   <class>Codethink::lvtqtw::ErrorView</class>
   <extends>QWidget</extends>
   <header location="global">ct_lvtqtw_errorview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Codethink::lvtqtw::WelcomeScreen</class>
   <extends>QWidget</extends>
   <header location="global">ct_lvtqtw_welcomescreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
